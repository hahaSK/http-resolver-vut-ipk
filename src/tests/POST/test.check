www.fit.vutbr.cz:A=147.229.9.23
www.google.com:A=74.125.69.147
www.seznam.cz:A=77.75.75.176
147.229.14.131:PTR=dhcpz131.fit.vutbr.cz
ihned.cz:A=46.255.231.42