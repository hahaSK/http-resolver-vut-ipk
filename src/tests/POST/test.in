curl --data-binary @"./POST/queries.txt" -X POST http://localhost:5353/dns-query