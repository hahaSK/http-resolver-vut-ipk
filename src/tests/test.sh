declare -a get_arr=("test_domain" "test_ip")

method="GET"

for i in "${get_arr[@]}"
do
    rm "./$method/$i.out"
    touch "./$method/$i.out"
    echo "./$method/$i"
    cat "./$method/$i.in" | /bin/bash >> "./$method/$i.out"
    diff -Z -q "./$method/$i.out" "./$method/$i.check"
done

declare -a post_arr=("test")

method="POST"

for i in "${post_arr[@]}"
do
    rm "./$method/$i.out"
    touch "./$method/$i.out"
    echo "./$method/$i"
    cat "./$method/$i.in" | /bin/bash >> "./$method/$i.out"
    diff -Z -q "./$method/$i.out" "./$method/$i.check"
done