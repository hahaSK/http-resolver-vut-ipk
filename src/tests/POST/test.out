www.fit.vutbr.cz:A=147.229.9.23
www.google.com:A=172.217.23.196
www.seznam.cz:A=77.75.75.172
147.229.14.131:PTR=dhcpz131.fit.vutbr.cz
ihned.cz:A=46.255.231.42
